### PR TITLE
docker-compose fixes

### PR DESCRIPTION
**Pull request by oliverlevay**

### CODE REVIEW

File: docker-compose.deploy.yml [hunk 1]
```diff
@@ -111,6 +113,7 @@ services:
     env_file: .env
     networks:
       - local
+      - files
```
Comment (LucasBoberg) [inline docker-compose.deploy.yml:116]: Why is this needed?

**Review by LucasBoberg — APPROVED** [review]

File: docker-compose.deploy.yml (hunk 1 above)
Comment (oliverlevay) [reply docker-compose.deploy.yml:116]: to fix meeting documents
